rfc/2/index.md

```diff
@@ -0,0 +1,459 @@
+- What related issues do you consider out of scope for this RFC that could be
+  addressed in the future independently of the solution that comes out of this
+  RFC?
+-->
+
+## Performance
+
+The adoption of Zarr v3 will not necessarily have an impact on performance.
+The performance is determined by a wide range of parameters, many of which are specific to implementations.
+
+Using sharding can have a profound impact on the number of files/objects that large images consume.
+On some storage backends using less files/objects can be beneficial for the performance of various operations.
+In particular, the chunk sizes can be made small to facilitate interactive visualization without incurring the overhead of too many files/objects.
+
+## Backwards Compatibility
+
+The metadata of Zarr v3 arrays are not backwards compatible with Zarr v2.
+
+It is RECOMMENDED that implementations of OME-Zarr specify the version of the OME-Zarr specification that they support.
+
+It is RECOMMENDED that implementations of OME-Zarr that support both v2 and v3-based OME-Zarr versions auto-detect the underlying Zarr version.
+
+While the metadata of Zarr v3 is not backwards compatible, the chunk data is largely backwards compatible, only depending on compressor configuration.
+[There are scripts available](https://github.com/scalableminds/zarrita/blob/8155761/zarrita/array_v2.py#L452-L559) to migrate Zarr v2 metadata to Zarr v3.
+This is generally a light-weight operation.
+Zarr v3 and v2 metadata can exist side-by-side within a Zarr hierarchy.
+
+## Abandoned Ideas
+
+Previous versions of this proposal contained changes to referencing `labels` in the OME-Zarr metadata.
+This has been delayed to future RFCs.
+
+## Examples
+
+File hierarchy of one multi-scale OME-Zarr image `456.zarr`:
+
+```
+456.zarr
+│
+├── zarr.json
+├── 1
+│   ├── zarr.json
+│   └─ c
+│      ├─ 0
+│      |  ├─ 0
+│      |  |  ├─ 0
+│      |  |  |  ├─ 0
+│      |  |  |  └─ ...
+│      |  |  └─ ...
+│      |  └─ ...
+│      └─ ...
+│   ...
+└── n
+```
+
+`456.zarr/zarr.json`:
+
+```json
+{
+  "zarr_format": 3,
+  "node_type": "group",
+  "attributes": {
+    "https://ngff.openmicroscopy.org/0.6": {
+      "multiscales": [
+        {
+          "coordinateSystems": [
+            {
+              "name": "root",
+              "axes": [
+                {
+                  "name": "c",
+                  "type": "channel",
+                  "discrete": true
+                },
+                {
+                  "name": "x",
+                  "type": "space",
+                  "unit": "nanometer"
+                },
+                {
+                  "name": "y",
+                  "type": "space",
+                  "unit": "nanometer"
+                },
+                {
+                  "name": "z",
+                  "type": "space",
+                  "unit": "nanometer"
+                }
+              ]
+            }
+          ],
+
+          "datasets": [
+            {
```

Comment:
The current (v0.4) version specifies that image paths `MUST` be alphanumeric. Is it within the scope of this RFC to relax that constraint?

https://github.com/ome/ngff/blob/b43000c65fd4a0cde67a2f32da447deb856b774b/0.4/index.bs#L582-L588

Reply:
Sorry I misread this to be the HCS paths...